### PR TITLE
Add cloud cost policy

### PR DESCRIPTION
**Pull request by daxmc99**
Adds cloud cost policy for work
https://github.com/sourcegraph/customer/issues/1176

### CODE REVIEW

File: content/departments/cloud/cloud-cost.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Requirements
+
+Utilize **feature flags** (ie through site-config or experimental flags) such that a feature can be enabled or disabled as needed. This also allows for the Cloud team to roll out the changes using “canary” methodologies to limit the blast radius for managed instances.
+
+Monitor and test features on S2. S2 has continous deployment so changes that are pushed to the `main` branch will be deployed to S2. Its not always possible to determine how a change may affect the overall cost. It is the feature team’s responsibility to check the billing page to ensure that their feature is not causing any large spikes in cost. The billing view is saved [here](https://console.cloud.google.com/billing/017005-C370B2-0E3030/reports;savedView=8549f3e5-8bfa-4a88-9b41-42a72eeb93f5?organizationId=244397465763&project=sourcegraph-dev).
+
+File a “Cloud Launch” ticket on our customer repository
+Post in the #cloud channel when the issue has been created
+Label the issue `cloud-feature`
+The Cloud team has an SLA of 7 days to respond to the ticket with questions.
```
Comment (michaellzc) [inline content/departments/cloud/cloud-cost.md]: I suppose there will be a issue template? is it still WIP?

cc @rafax

File: content/departments/cloud/cloud-cost.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Infrastructure costs
+
+One of the original goals of Cloud is to keep infrastructure costs capped to no more than 10% of a given customer’s ARR (private in handbook). This expresses the relationship between increased customer usage and infrastructure cost.
```
Comment (michaellzc) [inline content/departments/cloud/cloud-cost.md]: should we add a link to our billing dashboard?

ideally a saved view of all MI cost or maybe instruction of how to navigate around the billing dashboard

File: content/departments/cloud/cloud-cost.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Feature issue:
+Team owner:
+
+Possible cost increase: Yes/No
+If yes, estimated cost increase per month (we have billing calc here):
+If yes, what components do you expect to have increased cost:
+Has this been rolled out on S2 yet?:
+
+Any additional information we should know?
```
Comment (michaellzc) [inline content/departments/cloud/cloud-cost.md]: it feels off the template is here in the handbook. should it be moved to the customer repo?

File: content/departments/cloud/cloud-cost.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## “Business” size
+
+<!--
+Our standard size estimate is here (this is dynamic calculator). These are fixed costs.
+With networking and executors (which can vary in cost) we attempt to keep the overall cost around $500 (private in handbook) per month. Any changes that represent an additional fixed cost must obtain approval from the Cloud team (ie an additional database or storage solution). This is needed to ensure that new features do not endanger our standard pricing model. This pricing is also used for trial instances which is why it's imperative that it does not change without approval. -->
```
Comment (michaellzc) [inline content/departments/cloud/cloud-cost.md]: is this commented out on purpose?

File: content/departments/cloud/cloud-cost.md (hunk 1 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: We have a google doc template for now. Will update in the future if we decide to leverage a github issue template

File: content/departments/cloud/cloud-cost.md (hunk 4 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Removed

File: content/departments/cloud/cloud-cost.md (hunk 3 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Removed, I linked to the Cloud Launch google doc. Remains to be seen if that should be a github issue template

File: content/departments/cloud/cloud-cost.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Utilize **feature flags** (ie through site-config or experimental flags) such that a feature can be enabled or disabled as needed. This also allows for the Cloud team to roll out the changes using “canary” methodologies to limit the blast radius for managed instances.
+
+Monitor and test features on S2. S2 has continous deployment so changes that are pushed to the `main` branch will be deployed to S2. Its not always possible to determine how a change may affect the overall cost. It is the feature team’s responsibility to check the billing page to ensure that their feature is not causing any large spikes in cost. The billing view is saved [here](https://console.cloud.google.com/billing/017005-C370B2-0E3030/reports;savedView=8549f3e5-8bfa-4a88-9b41-42a72eeb93f5?organizationId=244397465763&project=sourcegraph-dev).
+
+File a “Cloud Launch” ticket on our customer repository using the
+[Cloud Launch template](https://docs.google.com/document/d/1oE2PJSdgqcX3ZRApWXwabgDtzFK4-0PZ3js5PTxsavw/edit?usp=sharing)
+
+Post in the #cloud channel when the issue has been created
+Label the issue `cloud-feature`
+The Cloud team has an SLA of 7 days to respond to the ticket with questions.
```
Comment (filiphaftek) [inline content/departments/cloud/cloud-cost.md]: I would propose to do it the same as with trials, so place this into SLA table for Cloud and link.

File: content/departments/cloud/cloud-cost.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- Complexity (is this feature still being tested and requires additional steps)
+
+If any of the above is true, this shouldn’t prevent you from launching on Cloud as long as your team will work closely with ours and this feature is clearly flagged as experimental.
+
+# Requirements
+
+Utilize **feature flags** (ie through site-config or experimental flags) such that a feature can be enabled or disabled as needed. This also allows for the Cloud team to roll out the changes using “canary” methodologies to limit the blast radius for managed instances.
+
+Monitor and test features on S2. S2 has continous deployment so changes that are pushed to the `main` branch will be deployed to S2. Its not always possible to determine how a change may affect the overall cost. It is the feature team’s responsibility to check the billing page to ensure that their feature is not causing any large spikes in cost. The billing view is saved [here](https://console.cloud.google.com/billing/017005-C370B2-0E3030/reports;savedView=8549f3e5-8bfa-4a88-9b41-42a72eeb93f5?organizationId=244397465763&project=sourcegraph-dev).
```
Comment (filiphaftek) [inline content/departments/cloud/cloud-cost.md]: I would link to s2 explanation or maybe at least URL? b/c I see often confusion between dotcom and s2

Comment (filiphaftek) [inline content/departments/cloud/cloud-cost.md]: I would explicitly mention to check Grafana i.e for errors or [excessive usage of resources](https://sourcegraph.sourcegraph.com/-/debug/grafana/d/containers/global-containers-resource-usage?orgId=1) and compare to last 2 weeks.

**Review by filiphaftek — APPROVED** [review]
LGTM 👍  left some minor comments

File: content/departments/cloud/cloud-cost.md (hunk 6 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Added a direct link

Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Added a direct link to Grafana

File: content/departments/cloud/cloud-cost.md (hunk 5 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Clarified 24 hour SLA

File: content/departments/cloud/cloud-cost.md (hunk 2 above)
Comment (daxmc99) [reply content/departments/cloud/cloud-cost.md]: Linked to dashboard

**Pull request merged**